tool/lib/release_note_classes.g.dart

```diff
@@ -0,0 +1,62 @@
+// GENERATED CODE - DO NOT MODIFY BY HAND
+
+part of 'release_note_classes.dart';
```

Comment:
just to be clear, all of this is generated by the json_annotation package? will we ever have to regenerate this, and if so is that a manual or automatic process?

Reply:
Great question,
This file is genereated when we run build runner:
`dart run build_runner build --delete-conflicting-outputs`

It only needs to be updated when the release_note_classes file is changed.

I'll add a comment to the top of the file for our future selves, reminding about the build command